src/Actions/Family/CreateFamily.php

```diff
@@ -31,7 +32,8 @@ public function __construct(EntityManagerInterface $entityManager)
 
     public function __invoke(ServerRequestInterface $request, ResponseInterface $response)
     {
-        $family = $request->getAttribute('familyEntity');
+        /** @var \Choredo\Entities\Family $family */
+        $family = $request->getAttribute(REQUEST_RESOURCE);
```

Comment:
Is it worth asserting that this is what you think it is?

Reply:
The inline docblock? I just like to get the sweet, sweet autocomplete in my IDE :o